File: Ser2023/Company/Company.Web/Modules/ServerTypes/Company/BusinessEntityContactForm.ts
import { StringEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface BusinessEntityContactForm {
    PersonId: StringEditor;
    ContactTypeId: StringEditor;
}

export class BusinessEntityContactForm extends PrefixedContext {
    static formKey = 'Company.BusinessEntityContact';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!BusinessEntityContactForm.init)  {
            BusinessEntityContactForm.init = true;

            var w0 = StringEditor;

            initFormType(BusinessEntityContactForm, [
                'PersonId', w0,
                'ContactTypeId', w0
            ]);
        }
    }
}

File: Ser2025/Company/Company.Web/Modules/Company/BusinessEntityContact/BusinessEntityContactForm.cs
using Serenity.ComponentModel;

namespace Company.Company.Forms;

[FormScript("Company.BusinessEntityContact")]
[BasedOnRow(typeof(BusinessEntityContactRow), CheckNames = true)]
public class BusinessEntityContactForm
{
    public int BusinessEntityId {  get; set; }
    public int PersonId { get; set; }
    public int ContactTypeId { get; set; }
}